tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -1019,6 +1066,16 @@ const reducer = createReducer(
           end: {step: state.stepMinMax.max},
         };
       }
+      Array.from(nextCardToTimeSelection.entries()).forEach(
```

Comment:
Similarly, you need to set end: null for each time selection when you turn off range selection. You don't necessarily need to do that in this PR but keep it in mind.